Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml

```diff
@@ -0,0 +1,181 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Recorded Future Feed
+  version: -1
+configuration:
+- defaultvalue: A - Completely reliable
+  display: Source reliability
+  name: feedInstanceReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- defaultvalue: suddenDeath
+  hidden: true
+  name: expirationPolicy
+  required: false
+  type: 0
+- hidden: true
+  name: expirationInterval
+  required: false
+  type: 1
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation.
+  defaultvalue: feedInstanceReputationNotSet
+  display: Indicator reputation
+  name: feedInstanceReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- defaultvalue: '60'
+  display: Fetch Interval
+  name: feedFetchInterval
+  required: true
+  type: 19
```

Comment:
You also need to include the `feedBypassExclusionList` parameter. Look at the confluence [page](https://confluence.paloaltonetworks.com/pages/viewpage.action?spaceKey=DemistoContent&title=Minemeld+miners+to+Demisto+feed+integration) towards the bottom to see the updated server parameter values. Add any you are missing, please. It might be best to follow the order given there as well.

Reply:
Done